### DISCUSSION TITLE
Consider renaming `snapshot.name_template` to `snapshot.version_template`?

### DISCUSSION THREAD

**Discussion by stdedos** [goreleaser/goreleaser#5016] [Q&A] [answered]
Can `snapshot.name_template` be renamed to ... `version_template`, or something like that?

(If my assumption https://github.com/orgs/goreleaser/discussions/5015 is correct ofc)

`name_template` sounds to me like it's going to change the `name`. It seems you are changing the `version`.

**Comment by caarlos0** [comment, marked as answer]
yeah I think that makes sense

see https://github.com/goreleaser/goreleaser/issues/5017
